go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	k8s.io/api v0.19.3
 	k8s.io/apiextensions-apiserver v0.19.3
 	k8s.io/apimachinery v0.19.3
+	k8s.io/autoscaler/vertical-pod-autoscaler v0.9.0
```

Comment:
We are currently running 0.8.0. Probably should align this.

Reply:
So, question is do we not want to upgrade to 0.9.0? Reason I'm asking is 0.9.0 added possibility of disabling scaling for individual containers, and I wanted to see if this is something we need with Prometheus.

I'm thinking, Prometheus has 3 containers, and we don't necessarily want to have evictions/re-deployments because of any of the `*-reloader` containers, so we should probably exclude them. I was just going to see if this happens in practice.

Reply:
OK I just checked that and it does indeed apply recommendations to all containers currently:

![testing-5 godsmack-prometheus per-container-req-limits 2020-11-30--12-05-02](https://user-images.githubusercontent.com/42101611/100608591-88075600-3304-11eb-809c-57289ebd54b7.png)

WDYT, should we exclude the reloader containers?

Reply:
Is is possible to enable recommended mode for the reloader and auto for hte prometheus pod? Otherwise, I would agree we should exclude them as they do not really fluctuate

Reply:
No, per-container you can only set it to auto(on)/off.

Reply:
I think off is recommended mode but I would ignore those reloaders

Reply:
Done.